discovery-server/src/test/java/com/discovery/discoveryserver/DiscoveryServerApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.discovery.discoveryserver;
```

Comment:
Discovery Server files are not needed since those well known service from Netflix. No need to commit these things instead test it locally and commit only the DiscoveryClient changes